client/state/action-types.js

```diff
@@ -17,6 +17,8 @@ export const EDITOR_CONTACT_FORM_ADD_DEFAULT_FIELD = 'EDITOR_CONTACT_FORM_ADD_DE
 export const EDITOR_CONTACT_FORM_CLEAR_FORM = 'EDITOR_CONTACT_FORM_CLEAR_FORM';
 export const EDITOR_CONTACT_FORM_LOAD_FORM = 'EDITOR_CONTACT_FORM_LOAD_FORM';
 export const EDITOR_CONTACT_FORM_REMOVE_FIELD = 'EDITOR_CONTACT_FORM_REMOVE_FIELD';
+export const EDITOR_CONTACT_FORM_UPDATE_FIELD = 'EDITOR_CONTACT_FORM_UPDATE_FIELD';
+export const EDITOR_CONTACT_FORM_UPDATE_SETTINGS = 'EDITOR_CONTACT_FORM_UPDATE_SETTINGS';
```

Comment:
As per https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#actions we have a preference for the verb to be suffixed on action names. Related actions should probably be renamed at the same time, so I'm OK with it not making it into this PR.

So `EDITOR_CONTACT_FORM_FIELD_UPDATE` vs `EDITOR_CONTACT_FORM_UPDATE_FIELD`